chitterView.js

```diff
@@ -0,0 +1,35 @@
+const ChitterModel = require('./chitterModel')
+const ChitterApi = require('./chitterApi')
+
+class ChitterView {
+  constructor(model, api) {
+    this.model = model;
+    this.api = api;
+
+    this.mainContainerEl = document.querySelector('#main-container');
+    this.buttonEl = document.querySelector('#show-peeps-button');
+ 
+    // this.buttonEl.addEventListener('click', () => {
+      this.api.getChitterData(data => {
+        console.log(data)
+        console.log(this.model.getPeeps())
+      });
+    // }); button to only show all peeps when clicked
```

Comment:
This is also happening in the index.js. Is it for a refresh button? 
You could try using this to begin setting up your add button. So every time you add it calls `display()` again. Even if it is not currently making a post request to the server.